diff --git a/media/libstagefright/omx/OMXNodeInstance.cpp b/media/libstagefright/omx/OMXNodeInstance.cpp
index f1365c6..efe3b90 100644
--- a/media/libstagefright/omx/OMXNodeInstance.cpp
+++ b/media/libstagefright/omx/OMXNodeInstance.cpp
@@ -1271,7 +1278,7 @@ status_t OMXNodeInstance::allocateSecureBuffer(
     if (mSecureBufferType[portIndex] == kSecureBufferTypeUnknown) {
         ALOGE("b/63522818");
         android_errorWriteLog(0x534e4554, "63522818");
-        return ERROR_UNSUPPORTED;
+        //return ERROR_UNSUPPORTED;
     }
 
     BufferMeta *buffer_meta = new BufferMeta(size, portIndex);
@@ -1338,7 +1345,7 @@ status_t OMXNodeInstance::allocateBufferWithBackup(
     if (mSecureBufferType[portIndex] != kSecureBufferTypeUnknown) {
         ALOGE("b/63522818");
         android_errorWriteLog(0x534e4554, "63522818");
-        return ERROR_UNSUPPORTED;
+        //return ERROR_UNSUPPORTED;
     }
 
     // metadata buffers are not connected cross process; only copy if not meta
